ubuntu/gpu/README.md

```diff
@@ -1,10 +1,10 @@
 # GPU Containers
 
-Example base layers to build your own container:
+Here are example base layers you can use to build a container. These cannot be directly used to launch a cluster -- you must first add a layer that creates a conda environment.
```

Comment:
This is not true for non-Python users. And the previous message already says "to build your own container".

Reply:
Right, thanks for clarifying. I will close this PR.